go/app/server.go

```diff
@@ -42,8 +46,10 @@ func (s Server) Run() int {
 	// set up routes
 	mux := http.NewServeMux()
 	mux.HandleFunc("GET /", h.Hello)
+	mux.HandleFunc("GET /items", h.GetItem)
```

Comment:
[must]
対応させるメソッドがズレているので、修正お願いします :pray:

```suggestion
	mux.HandleFunc("GET /items", h.GetItems)
```